### DISCUSSION TITLE
APN ip ranges

### DISCUSSION THREAD

**Discussion by abouillot-stl** [herlesupreeth/docker_open5gs#153] [Q&A] [answered]
Hello,

Thanks @herlesupreeth for the great job of putting this together.

I noticed in `smf.yaml`, `upf.yaml`, `upf_init.sh,` `tun_if.sh` , `rtpengine_init.sh` and `rtpengine_init.sh` reference to networks IP `192.168.100.0/24` and `192.168.101.0/24`.

Couldn't those collide with the network used by the physical connection of the host PC ?

In case the Docker host PC IP is also in the `192.168.100.0/24` network, sould I change these networks IPs?

Thanks,

**Comment by herlesupreeth** [comment, marked as answer]
I don't think it will affect because the packets from UPF will be NATed towards the internet and for IMS traffic it will remain inside docker network. But if you face issue send me a wireshark trace and I can take a look

**Comment by abouillot-stl** [comment]
Thanks for the confirmation. I might do it manually for the day, but looking forward for the official support.
